dsds.py:25,15:24:04:start:18:0ablag -- start
get_file:25,15:24:04:loadFiles:266:0ablag -- Start loadFiles, 527e4afc-4598-400f-8536-afa5324f0ba4 1ytJocI9f4gvmpnwLNNfpQzRPhcFs5EzR
requests:25,15:24:04:__call__:117:0ablag -- Making request: POST https://oauth2.googleapis.com/token
connecti:25,15:24:04:_new_conn:817:0ablag -- Starting new HTTPS connection (1): oauth2.googleapis.com
connecti:25,15:24:04:_make_request:393:0ablag -- https://oauth2.googleapis.com:443 "POST /token HTTP/1.1" 200 None
selector:25,15:24:04:__init__:59:0ablag -- Using selector: EpollSelector
test_uti:25,15:24:04:print_size:82:0ablag -- ---------------Memory usage: 87100 (kb)---------------0 mins since start
test_uti:25,15:24:04:print_size:100:0ablag -- /home/henry/pydocs/data/527e4afc-4598-400f-8536-afa5324f0ba4/
0/0 at 2020-04-25 15:24:04.209426
dumped: 0
get_file:25,15:24:04:getIdsRecursive:52:0ablag -- getIdsRecursive looping
get_file:25,15:24:04:getIdsRecursive:52:0ablag -- getIdsRecursive looping
get_file:25,15:24:04:getIdsRecursive:52:0ablag -- getIdsRecursive looping
test_uti:25,15:24:04:tryGetQueue:226:0ablag -- getIdswaiting 1 1
test_uti:25,15:24:04:tryGetQueue:226:0ablag -- getIdswaiting 1 1
test_uti:25,15:24:05:print_size:82:0ablag -- ---------------Memory usage: 87300 (kb)---------------0 mins since start
test_uti:25,15:24:05:print_size:100:0ablag -- /home/henry/pydocs/data/527e4afc-4598-400f-8536-afa5324f0ba4/
0/0 at 2020-04-25 15:24:05.233512
dumped: 0
get_file:25,15:24:05:getIdsRecursive:52:0ablag -- getIdsRecursive looping
test_uti:25,15:24:05:tryGetQueue:226:0ablag -- getIdswaiting 1 1
get_file:25,15:24:05:getRevision:163:0ablag -- getRevision looping
gdoc.py:25,15:24:05:async_init:24:0ablag -- async init called
gdoc.py:25,15:24:06:get_last_revision:54:0ablag -- last revision id 546
gdoc.py:25,15:24:06:async_init:31:0ablag -- after await 546
test_uti:25,15:24:06:print_size:82:0ablag -- ---------------Memory usage: 87300 (kb)---------------0 mins since start
test_uti:25,15:24:06:print_size:100:0ablag -- /home/henry/pydocs/data/527e4afc-4598-400f-8536-afa5324f0ba4/
0/2 at 2020-04-25 15:24:06.255950
dumped: 0
get_file:25,15:24:06:getRevision:187:0ablag -- google doc
get_file:25,15:24:06:getRevision:163:0ablag -- getRevision looping
gdoc.py:25,15:24:06:async_init:24:0ablag -- async init called
gdoc.py:25,15:24:06:get_last_revision:54:0ablag -- last revision id 2502
gdoc.py:25,15:24:06:async_init:31:0ablag -- after await 2502
get_file:25,15:24:07:getRevision:187:0ablag -- google doc
get_file:25,15:24:07:getRevision:163:0ablag -- getRevision looping
test_uti:25,15:24:07:print_size:82:0ablag -- ---------------Memory usage: 94220 (kb)---------------0 mins since start
test_uti:25,15:24:07:print_size:100:0ablag -- /home/henry/pydocs/data/527e4afc-4598-400f-8536-afa5324f0ba4/
2/3 at 2020-04-25 15:24:07.279488
dumped: 0
get_file:25,15:24:07:getRevision:163:0ablag -- getRevision looping
test_uti:25,15:24:08:print_size:82:0ablag -- ---------------Memory usage: 94220 (kb)---------------0 mins since start
test_uti:25,15:24:08:print_size:100:0ablag -- /home/henry/pydocs/data/527e4afc-4598-400f-8536-afa5324f0ba4/
2/3 at 2020-04-25 15:24:08.304653
dumped: 0
get_file:25,15:24:08:getRevision:163:0ablag -- getRevision looping
configlo:25,15:24:09:handle_exception:81:0ablag -- Custom Uncaught exception
Traceback (most recent call last):
  File "dsds.py", line 37, in <module>
    start()
  File "dsds.py", line 34, in start
    loadFiles(uid, workingPath, fileid, creds)
  File "/home/henry/pydocs/processing/get_files.py", line 291, in loadFiles
    loop.run_until_complete(asyncio.gather(startTask))
  File "/usr/lib/python3.8/asyncio/base_events.py", line 603, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 570, in run_forever
    self._run_once()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 1823, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.8/selectors.py", line 468, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
KeyboardInterrupt
